sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/feedranges/FeedRangeEpkImpl.java

```diff
@@ -39,6 +39,9 @@
 
     public FeedRangeEpkImpl(final Range<String> range) {
         checkNotNull(range, "Argument 'range' must not be null");
+        if (range.getMin().compareTo(range.getMax()) > 1) {
```

Comment:
shouldn't this be `>0` ?

```suggestion
        if (range.getMin().compareTo(range.getMax()) > 0) {
```

Reply:
Yes - interesting - the exception was actually thrown - but i guess that was just luck